Establish positive data types for volume and size

gjtorikian opened the pull request:
This is a first pass run through to fix https://github.com/alpacahq/alpaca-trade-api-csharp/issues/283. Familiarize yourself with [Dot Net Data Types](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/data-types/) if it's helpful!

In several places, the signed Int64 data type is used, when a UInt64 data type could, and should, be used. For example:

* Volume in a trade
* Bid/ask sizes

This PR converts those variables to use UInt64. I truly considered whether UInt64 was the right choice, because it's upper boundary is an insane 1.8...E+19. UInt32 (upper bound 4,294,967,295) might be a more realistic choice.

Similarly, in at least one case (`Multiplier`), it _did_ make sense to drop the size completely. Multiplier [only has three values](https://alpaca.markets/docs/api-documentation/api-v2/account/) currently: 1, 2, and 4. Perhaps it would also make sense as an enum.

There is one more PR I'd like to make. Several utilities, like this one, also have quite large data types:

https://github.com/alpacahq/alpaca-trade-api-csharp/blob/353dc450efff6e24ba972a823dcc30277bddbdab/Alpaca.Markets/Helpers/DateTimeHelper.cs#L8-L11

It should be noted that currently, quantities _are_ kept at Int64; this is because negative values can be used when placing short orders:

https://github.com/alpacahq/alpaca-trade-api-csharp/blob/3dbbbfe55afef6fe95f9bca7df7ef0cf552272c1/UsageExamples/MeanReversionBrokerage.cs#L196-L203

OlegRa commented on Alpaca.Markets/Interfaces/IHistoricalItems.cs:
```diff
@@ -36,12 +36,12 @@ public interface IHistoricalItems<out TItem>
         /// Polygon v2 API only.
         /// </summary>
-        Int64 QueryCount { get; }
+        UInt64 QueryCount { get; }
```
Rollback changes in this interface and its implementation - both will be removed from SDK soon.

OlegRa reviewed (changes requested):
Thanks a lot, @gjtorikian for this work! In general, I'm OK with most of the proposed changes - just some minor tweaks are required. Please, review my comments for more details.

gjtorikian commented:
Ok @OlegRa, I believe I addressed the concerns.

Note that an entire interface must be marked as CLS-off; individual members cannot be marked that way.
